thanos/values.yaml

```diff
@@ -15,8 +15,10 @@ s3:
   enabled: false
   bucket: ""
   endpoint: ""
+  accessKey: ""
+  secretKey: ""
   insecure: false
-  signatureV2: true
+  signatureV2: false
```

Comment:
do we want to upstream this change?

Reply:
Yes, we do, I've described this in the PR description.
```
SignatureV2 value changed to match default for s3 according to documentation (https://thanos.io/storage.md/#aws-s3-configuration). AWS requires signature v4, and v2 here is required only if we use some other s3-compatible storage.
```

Reply:
If we leave signature v2, then it would be correct to rename `s3` section to something like `objectStorage`.